test/serializer/abstract/ResidualInvariant3.js

```diff
@@ -0,0 +1,13 @@
+function checker() {
+  let x = global.__abstract ? global.__abstract("number", "5") : 5;
+  global.__invariant && global.__invariant(x !== 5);
+}
+
+inspect = () => {
+  try {
+    checker();
+    return "Invariant violatedok";
```

Comment:
This value is a bit mysterious and makes the test case harder to read than is necessary.